optimize/backend/src/main/java/io/camunda/optimize/service/db/util/AggregationNameUtil.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Comment doesn't concern this file, but was this an active decision to not refactor such files e.g. removing `.../interpreter/util/AggregationResultMappingUtilES.java` and moving some of its methods to other classes? Or was it just overlooked. I'm fine either way, I personally don't think it's worth spending too much time refactoring an old branch, so happy to leave as is. Just want to make sure it is an active decision

Reply:
Because 8.6 doesn't have all the same files (it is kind of an interim OS and ES client state), it didn't make sense to do it. Given that those changes were really just boyscouting, I left them out here as an active decision